Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "warframe"
-version = "5.0.0"
+version = "5.0.1"
```

Comment:
isn't this already in master? looks like you've got some excess commits. probably need to rebase